recipes-multimedia/aamp/aampabr_git.bb

```diff
@@ -8,7 +8,7 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
 PV = "0.1.gitr${SRCPV}"
 
 SRC_URI = "git://github.com/rdkcmf/rdk-aampabr.git;branch=stable2;protocol=https"
```

Comment:
Can you please add BRANCH variable and make a weak assignment for SRCREV and BRANCH?